### ISSUE TITLE
Fix AMBER parser to use units consistent with the rest of the code

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/alchemical-analysis#52]
See #45 , @halx : The output units are inconsistent. For example, output shows:

```
The gradients from the correlated samples (kcal/mol):
0.00000 -15.817
0.10000 -24.593
0.20000 -32.826
0.30000 -44.704
0.40000 -56.149
0.50000 -61.926
0.60000 -65.991
0.70000 -77.821
0.80000 -103.103
0.90000 -134.217
1.00000 -125.538
TI = -67.201
```

but later

```
Estimating the free energy change with TI...

States TI (kJ/mol)

0 -- 1 -8.454 +- 0.333 
1 -- 2 -12.068 +- 0.506 
2 -- 3 -16.160 +- 0.597 
3 -- 4 -20.983 +- 0.569 
4 -- 5 -24.701 +- 0.445 
5 -- 6 -26.775 +- 0.432 
6 -- 7 -30.216 +- 0.564 
7 -- 8 -37.965 +- 0.551 
8 -- 9 -49.648 +- 0.448 
9 -- 10 -54.341 +- 0.337

TOTAL:     -281.312  +-  2.176 
```
Using two different sets of units is confusing and can easily cause the user to think the results are inconsistent.

**Comment by halx** [comment]
Quick fix in my branch but needs a better, centralised solution.

**Comment by davidlmobley** [comment]
@halx - do a PR with this for the time being, while waiting for the centralized solution?

**Comment by halx** [comment]
Fixed and PR.

**Comment by davidlmobley** [comment]
Resolved by #61
